sdk/nodejs/dagger/api/types.ts

```diff
@@ -0,0 +1,534 @@
+// THIS FILE HAS BEEN GENERATED WITH GET-GRAPHQL-SCHEMA
```

Comment:
Same question about this file vs `api/types.ts`

Reply:
same, old files that somehow stayed.